src/neo/IO/Caching/DataCache.cs

```diff
@@ -204,8 +211,11 @@ public IEnumerable<Trackable> GetChangeSet()
         {
             lock (dictionary)
             {
-                foreach (Trackable trackable in dictionary.Values.Where(p => p.State != TrackState.None))
-                    yield return trackable;
+                foreach (TKey key in fuzzyChangeSet)
+                {
+                    if (dictionary.ContainsKey(key))
```

Comment:
Is it possible for this to be `false`?

Reply:
It's possible, when ADD and DELETE. 
https://github.com/neo-project/neo/blob/a1542f13bb444db3b6c31956f434da54f06aff22/src/neo/IO/Caching/DataCache.cs#L110-L118

I think we can also remove from it from `fuzzyChangeSet`.

Reply:
Agree. We should remove it from `fuzzyChangeSet`.